packages/cli/test/integration/shared/testDb.ts

```diff
@@ -124,57 +82,16 @@ export async function truncate(names: Array<(typeof repositories)[number]>) {
 	}
 }
 
-// ----------------------------------
-//        connection options
-// ----------------------------------
-
 /**
- * Generate options for an in-memory sqlite database connection,
- * one per test suite run.
+ * Generate options for a bootstrap DB connection, to create and drop test databases.
  */
-const getSqliteOptions = ({ name }: { name: string }): ConnectionOptions => {
+export const getBootstrapDBOptions = (dbType: 'postgresdb' | 'mysqldb'): DataSourceOptions => {
+	const type = dbType === 'postgresdb' ? 'postgres' : 'mysql';
```

Comment:
We don't run these tests against SQLite? Or are they handled somehow differently?